README.md

```diff
@@ -1,96 +1,155 @@
+To get started, you'll need to create a [free Auth0 account](https://auth0.com/signup) and register an [Application](https://auth0.com/docs/applications).
+
+### Authentication API Client
+
+Begin by instantiating the SDK and passing the relevant details from your Application's settings page:
 
 ```php
-// Instantiate the base Auth0 class.
+use Auth0\SDK\Auth0;
+
 $auth0 = new Auth0([
-	// The values below are found on the Application settings tab.
-	'domain' => 'your-tenant.auth0.com',
-	'client_id' => 'application_client_id',
-	'client_secret' => 'application_client_secret',
-
-	// This is your application URL that will be used to process the login.
-	// Save this URL in the "Allowed Callback URLs" field on the Application settings tab
-	'redirect_uri' => 'https://yourdomain.com/auth/callback',
+  // The values below are found on the Application settings tab.
+  'domain'        => '{YOUR_TENANT}.auth0.com',
+  'client_id'     => '{YOUR_APPLICATION_CLIENT_ID}',
+  'client_secret' => '{YOUR_APPLICATION_CLIENT_SECRET}',
+
+  // This is your application URL that will be used to process the login.
+  // Save this URL in the "Allowed Callback URLs" field on the Application settings tab
+  'redirect_uri' => 'https://{YOUR_APPLICATION_CALLBACK_URL}',
 ]);
 ```
 
-**The values above should not be hard-coded in a production application** but will suffice for testing or local development. Please see our complete guide on the [main documentation page](https://auth0.com/docs/libraries/auth0-php#getting-started) for more information on how to store and use these values.
+**Note:** _In a production application you should never hardcode these values. Consider using environment variables to store and pass these values to your application, as suggested in our [documentation](https://auth0.com/docs/libraries/auth0-php#getting-started)._
+
+Using the SDK, making requests to Auth0's endpoints couldn't be simpler. For example, signing users in using Auth0's [Universal Login](https://auth0.com/docs/universal-login) and retrieving user details can be done in a few lines of code:
+
+```php
+// Do we have an authenticated session available?
+if ($userinfo = $auth0->getUser()) {
+  // Output the authenticated user
+  var_dump($userinfo);
+  exit;
+}
+
+// No session was available, so redirect to Universal Login page
+$auth0->login();
+```
+
+Further examples of how you can use the Authentication API Client can be found on [our documentation site](https://auth0.com/docs/libraries/auth0-php/).
+
+### Management API Client
+
+This SDK also offers an interface for Auth0's Management API, using a management access token.
+
+Management access tokens can be generated using the `getManagementAccessToken` method and the Authentication API, please review our [documentation](https://auth0.com/docs/libraries/auth0-php/using-the-management-api-with-auth0-php) for guidance.
```

Comment:
It sounds like `getManagementAccessToken ` is part of the SDK, but it looks like it isn't. I also think its confusing that, when going to https://auth0.com/docs/libraries/auth0-php/using-the-management-api-with-auth0-php and then clicking through to https://auth0.com/docs/libraries/auth0-php/using-the-authentication-api-with-auth0-php, there is no mention anymore of `getManagementAccessToken `.

I think we might want to mention something along the lines of 

> This SDK also offers an interface for Auth0's Management API which, in order to access it, requires an Access Token that is issued specifically for your tenant's Management API by specifying the corresponding Audience.
>
> The process for retrieving such an Access Token is described in our [documentation](https://auth0.com/docs/libraries/auth0-php/using-the-management-api-with-auth0-php) .

Reply:
I like that much better, thanks for the suggestion